src/dymaptic.GeoBlazor.Core/Scripts/featureLayerView.ts

```diff
@@ -33,6 +33,9 @@ export default class FeatureLayerViewWrapper implements IPropertyWrapper {
         }
     }
 
+    unwrap() {
```

Comment:
saw this added on a number of these files, i needed to look this up

Reply:
It's my own method, although the pattern exists elsewhere. This is saying "I created this custom wrapper around the Esri type, give me back the Esri type".

Reply:
ah gotcha